### PR TITLE
Issue 13946 fix all by basetype

### PR DESCRIPTION
**Pull request by freddyucv**

### CODE REVIEW

File: dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/ajax/ContentletAjax.java [hunk 1]
```diff
@@ -454,6 +448,15 @@ public List searchContentlets(String structureInode, List<String> fields, List<S
 		        page, orderBy, perPage,currentUser, sess, modDateFrom, modDateTo);
 	}
 
+	public List searchContentlets(String[] structureInodes, List<String> fields, List<String> categories, boolean showDeleted,
+								  boolean filterSystemHost,  boolean filterUnpublish, boolean filterLocked, int page, int perPage,String orderBy, String modDateFrom,
+								  String modDateTo) throws DotStateException, DotDataException, DotSecurityException {
+		String structureInodesJoined = String.join(",", structureInodes);
```
Comment (jdotcms) [inline dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/ajax/ContentletAjax.java]: replace it by StringUtils.COMMA

File: dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/ajax/ContentletAjax.java [hunk 2]
```diff
@@ -537,12 +540,27 @@ public List searchContentletsByUser(String structureInode, List<String> fields,
 		List<Object> headers = new ArrayList<Object>();
 		Map<String, Field> fieldsMapping = new HashMap<String, Field>();
 		Structure st = null;
-		if(!Structure.STRUCTURE_TYPE_ALL.equals(structureInode)){
+		if(!Structure.STRUCTURE_TYPE_ALL.equals(structureInode) && structureInode.indexOf(',') == -1){
```
Comment (jdotcms) [inline dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/ajax/ContentletAjax.java]: StringUtils.COMMA

File: dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/ajax/ContentletAjax.java [hunk 3]
```diff
@@ -537,12 +540,27 @@ public List searchContentletsByUser(String structureInode, List<String> fields,
...
 		Map<String, Field> fieldsMapping = new HashMap<String, Field>();
 		Structure st = null;
-		if(!Structure.STRUCTURE_TYPE_ALL.equals(structureInode)){
+		if(!Structure.STRUCTURE_TYPE_ALL.equals(structureInode) && structureInode.indexOf(',') == -1){
 		    st = CacheLocator.getContentTypeCache().getStructureByInode(structureInode);
 		    lastSearchMap.put("structure", st);
 		    luceneQuery.append("+contentType:" + st.getVelocityVarName() + " ");
-		}
-		else {
+		} else if (!Structure.STRUCTURE_TYPE_ALL.equals(structureInode) && structureInode.indexOf(',') != -1) {
```
Comment (jdotcms) [inline dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/ajax/ContentletAjax.java]: StringUtils.COMMA

File: dotCMS/src/main/webapp/html/js/dotcms/dijit/form/ContentSelector.js [hunk 4]
```diff
@@ -659,10 +660,30 @@ dojo.declare("dotcms.dijit.form.ContentSelector", [dijit._Widget, dijit._Templat
 			this.currentSortBy = sortBy;
 		}
 
-		//ContentletAjax.searchContentlets (this.structureInode, fieldsValues, categoriesValues, false, false, this.currentPage, this.currentSortBy, null, null, false, dojo.hitch(this, this._fillResults));
-		ContentletAjax.searchContentlets(this.structureInode, fieldsValues, categoriesValues, false,
-		        false,  false, false, this.currentPage, 10,this.currentSortBy, null,
-		        null,dojo.hitch(this, this._fillResults));
+        //ContentletAjax.searchContentlets (this.structureInode, fieldsValues, categoriesValues, false, false, this.currentPage, this.currentSortBy, null, null, false, dojo.hitch(this, this._fillResults));
```
Comment (jgambarios) [inline dotCMS/src/main/webapp/html/js/dotcms/dijit/form/ContentSelector.js]: You can delete this line, if is not going to be use we should remove it.

**Review by jgambarios — CHANGES_REQUESTED** [review]

**Review by jgambarios — APPROVED** [review]

**Pull request merged**